website-next/components/common/env-vars.ts

```diff
@@ -44,9 +45,7 @@ export function isBackendBFF(): boolean {
 }
 
 export const UTOPIA_BACKEND =
-  (PRODUCTION_OR_STAGING_CONFIG
```

Comment:
The issue is that this is never `true` in the context of the website-next project. The correct fix is to pass through the environment variable